.ci/es-snapshots/Jenkinsfile_build_es

```diff
@@ -157,6 +157,9 @@ def buildArchives(destination) {
       ./gradlew -p distribution/archives assemble --parallel
       mkdir -p ${destination}
       find distribution/archives -type f \\( -name 'elasticsearch-*-*-*-*.tar.gz' -o -name 'elasticsearch-*-*-*-*.zip' \\) -not -path *no-jdk* -exec cp {} ${destination} \\;
+      ./gradlew -p distribution/docker assemble --parallel
```

Comment:
I am testing this - but I believe we replace/combine this line with the one in 157 with `./gradlew -Dbuild.docker=true assemble`

Reply:
I consolidated the docker assemble.